haiku-misc/yoshi/yoshi-1.1.recipe

```diff
@@ -0,0 +1,61 @@
+SUMMARY="Yab over the Shell Interface"
+DESCRIPTION="Yoshi is a tool for creating native HAIKU dialog from the shell"
+HOMEPAGE="http://software.besly.de/"
+COPYRIGHT=" 2016 - 2020 Jim Saxton, Fat Elk Software"
+LICENSE="yoshi"
+REVISION="1"
+SOURCE_URI="https://github.com/bbjimmy/Yoshi/archive/yoshi_1.1.tar.gz"
+CHECKSUM_SHA256="0652db14a9415558c7dbbc5063ecfd9d1bdb7fd8457bc41cca8a3c6a47982be7"
+SOURCE_DIR="Yoshi-yoshi_1.1"
+
+ADDITIONAL_FILES="yoshi_postinstall.sh"
+
+ARCHITECTURES="x86_gcc2 !x86 x86_64"
+
```

Comment:
No new line between ARCHITECTURES and DISABLE_SOURCE_PACKAGE

Reply:
done